modules/management/namespace-credential-provisioner/scripts/reconcile.sh

```
@@ -408,9 +408,15 @@ kubectl get namespaces -o json | jq -r '
   [[ -z "$project_id" ]] && continue
   is_system_namespace "$ns" && continue
   [[ "$role" == "network-namespace" ]] && continue
-  log "INIT namespace: ${ns} (project: ${project_id})"
-  if on_added_namespace "$ns" "$project_id"; then
+  local sa_name="harvester-cloud-provider-${ns}"
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)